pkgs/development/python-modules/pyhive-integration/default.nix

```
@@ -15,7 +15,7 @@
 
 buildPythonPackage rec {
   pname = "pyhive-integration";
-  version = "1.0.2";
+  version = "1.0.3";
   pyproject = true;
 
   disabled = pythonOlder "3.6";
```

```suggestion
  disabled = pythonOlder "3.10";
```
https://github.com/Pyhass/Pyhiveapi/blob/v1.0.3/setup.cfg#L25